halyard-deploy/src/main/resources/debian/install.sh

```diff
@@ -57,45 +55,47 @@ else
 fi
 
 function add_redis_apt_repository() {
-  add-apt-repository -y ppa:chris-lea/redis-server
+  # Only Ubuntu prior to 18.04 LTS requires this PPA
+  if [ "${DISTRIB_RELEASE%%.*}" -lt "18" ]; then
+    echo "Adding Redis PPA repository for Ubuntu version less than 18.04"
+    add-apt-repository -y ppa:chris-lea/redis-server
+  fi
 }
 
 function add_spinnaker_apt_repository() {
+  echo "Adding Spinnaker apt repository"
   REPOSITORY_HOST=$(echo $REPOSITORY_URL | cut -d/ -f3)
-  if [[ "$REPOSITORY_HOST" == "dl.bintray.com" ]]; then
```

Comment:
Bintray was sunset and is not coming back so removing this.
Anyone installing a Halyard version containing this change will automatically get the new spinnaker-community repos.